vic/drivers/shared_image/include/vic_driver_shared_image.h

```diff
@@ -204,5 +205,6 @@ void vic_restore(void);
 void vic_start(void);
 void vic_store(void);
 void vic_write(void);
+cv_converter*udunits_conversion(const char*const from, const char*const to);
```

Comment:
Yes I did and just tried it again (without succes). No I changed it by hand to: 
`cv_converter *udunits_conversion(const char *const from, const char *const to);`